docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
```

Comment:
Since many of these tools are not self-describing, I think we need a bulleted list with a short description of what each service _does_. Otherwise it's hard to compare capabilities between the 4 if you don't know what they are. (RPC is self describing, but how would you know what Horizon is?)

Reply:
out of curiosity - why aren't we mentioning CDP?

Reply:
Yeah I'll mention CDP in the form of Galexie

Reply:
updated in updated in a088c83120488cfcf7ef46830ab6621107828e2c